src/components/header/styled-header.js

```diff
@@ -0,0 +1,33 @@
+import styled from "styled-components";
```

Comment:
use header-styles.js instead of styled-header.js

Reply:
Ok.